### ISSUE TITLE
Overlapping Audio Cue

### ISSUE THREAD

**Issue by IronDome2** [ArduPilot/MissionPlanner#1811]
#### Issue details
Some audio cue overlap. Such as "Initializing APM", and "Bad GPS Health" are heard at the same time and during operations, "Heading to Way-point #" and "Battery ##Volt, ##%" 

I did not encounter this issue with previous versions.

Thanks in advance.
ID

#### Version
MissionPlanner-1.3.55.msi

#### Platform
[  ] All
[  ] AntennaTracker
[  ] Copter
[  ] Plane
[  ] Rover
[  ] Sub

#### Airframe type
_What type of airframe (flying wing, glider, hex, Y6, octa etc)_

#### Hardware type
_What autopilot hardware was used? (pixhawk, pixracer, PX4FMU etc)_

#### Logs
_Please provide a link to any relevant logs that show the issue_

**Comment by RHITNL** [comment]
Yes, I do have the exact same "problem" but I always check message tab as well
